content/applications/marketing/survey.rst

```diff
@@ -1,8 +1,286 @@
+  correct
+
+* **Multiple choice: multiple answers allowed** - of the multiple options offered, more than one
+  of them can be correct
+
+* **Matrix** - rows and columns are used to gauge certain things from participants on a scale (i.e.
+  *"On a scale from 1-10, how happy are you with your position?"*)
+
+*Answers* and *Options* tab (based on *Question Type*)
+------------------------------------------------------
+
+Each "Question Type" reveals different features/options in the *Answers* and *Options* tab that are
+specific to that unique "Question Type." However, the *Description* tab will always remain the same.
+
+Here's what the *Answers* tab offers for each individual "Question Type":
+
+* **Multiple Lines Text Box** - Nothing, because they want the participant to answer freely.
+
+* **Single Line Text Box** - *Input must be an email*, *Save as user nickname*, *Validate entry*
+
+* **Numerical Value** - *Validate entry*, *Correct answer* (numerical value), if the question
+  should be *Scored*, and if so, you can determine what participants *Score* with a correct response
+
+* **Date** - *Validate entry*, *Correct answer* (date), *Scored*, and *Score*
+
+* **Datetime** - *Validate entry*, *Correct answer* (date and time), *Scored*, and *Score*
+
+* **Multiple choice: only one answer** - *Choices* and the user selects one option as the correct
+  answer
+
+* **Multiple choice: multiple answers allowed** - *Choices* but the user can select more than one
+  option as the correct answer(s)
+
+* **Matrix** - *Choices* and *Rows*, wherein the *Rows* represent a certain question and the
+  *Choices* would represent various answer options to that question
+
+.. note::
+   The *Options* tab only looks different for: *Multiple Choice (One or Multiple answers)* and
+   *Matrix*. In those instances, the options are purely based on layout and answer options.
+
+Question configuration
+----------------------
+
+You can choose to validate an entry, make an answer mandatory, or activate "Conditional Display."
+
+.. image:: survey/survey_essentials/conditional-display.png
+   :align: center
+   :alt: conditional display activated in survey settings
+
+"Conditional Display"
+---------------------
+
+*Conditional Display* means a question is only displayed if the specified conditional answer has
+been selected in a previous question.
+
+After activating *Conditional Display*, you need to choose a *Triggering Question*. Once done, you
+should select the *Triggering Answer* linked to that question.
+
+.. image:: survey/survey_essentials/conditional-triggering-answers.png
+   :align: center
+   :alt: conditional triggering answer fields
+
+.. note::
+   When you are done, remember to hit *Save & Close*, which takes you back to the survey template
+   page. You can also hit *Save & New* to instantly start working on another question.
+
+Layout
+------
+
+This section primarily focuses on the overall presentation of your survey.
+
```

Comment:
Maybe add an H3 heading if you want this section to focus on the "Questions part" of the options